docs/cli-gui.md

```diff
@@ -0,0 +1,42 @@
+The AEA Command Line Interface (CLI) can also be invoked from a Graphical User Interface (GUI) which can be access from a web browser. 
+
+These instructions will take you through building an agent, starting an OEF Node and running the agent - all from the GUI. Once you can do this, the other operations should be fairly self-explanatory.
+
+## Preliminaries
+
+Ensure you have the framework installed and the CLI is working by following the quick-start guide. 
+
+## Starting the GUI
+Go to your working folder, where you want to create new agents. If you followed the quick start guide, this will be in the my_aea directory. Start the local web-server:
+``` bash
+aea gui
+```
+
+Open this page in a browser: http://127.0.0.1:8080
```

Comment:
minor fix: replace
`http://127.0.0.1:8080`
with 
`[http://127.0.0.1:8080](http://127.0.0.1:8080)`
or
`[localhost:8080](http://127.0.0.1:8080)`
or other variants.

The thing is to make the `http://127.0.0.1:8080` string a link.